.vscode/launch.json

```diff
@@ -70,7 +70,9 @@
         "ASPNETCORE_URLS": "http://localhost:5017",
         "ASPNETCORE_ENVIRONMENT": "Development",
         "KUBERNETESBICEPEXTENSIBILITYURL": "http://localhost:5017/api",
-        "RADIUSBACKENDURI": "http://localhost:9000"
+        "RADIUSBACKENDURI": "http://localhost:9000",
+        "KUBECONFIG": "${env:HOME}/.kube/config",
+        "RAD_LOCAL_DEVELOPMENT": "true"
```

Comment:
Where is this env variable (`RAD_LOCAL_DEVELOPMENT`) consumed?

Reply:
This was going to be used as a way to understand if a local development was in progress or not. After our discussion with @brooke-hamilton I decided to remove these added flags. Thanks for your review!